src/components/CurriculumComponents/CurriculumVisualiserFilters/CurriculumVisualiserFiltersMobile.tsx

```diff
@@ -0,0 +1,339 @@
+
+import { CurriculumVisualiserFiltersProps } from "./CurriculumVisualiserFilters";
+import { highlightedUnitCount } from "./helpers";
+
+import Box from "@/components/SharedComponents/Box";
+import Button from "@/components/SharedComponents/Button/Button";
+import ButtonGroup from "@/components/SharedComponents/ButtonGroup";
+import { getYearGroupTitle } from "@/utils/curriculum/formatting";
+import useAnalyticsPageProps from "@/hooks/useAnalyticsPageProps";
+import useAnalytics from "@/context/Analytics/useAnalytics";
+import { Thread } from "@/utils/curriculum/types";
+
+function StickyBit({
+  onOpenModal,
+  data,
+  trackingData,
+  selectedThread,
+  selectedYear,
+  yearSelection,
+  onSelectYear,
+}: Pick<
+  CurriculumVisualiserFiltersProps,
+  | "data"
+  | "trackingData"
+  | "selectedThread"
+  | "selectedYear"
+  | "yearSelection"
+  | "onSelectYear"
+> & { onOpenModal: () => void }) {
+  const { track } = useAnalytics();
+  const { analyticsUseCase } = useAnalyticsPageProps();
+
+  const { yearData, threadOptions, yearOptions } = data;
+
+  const highlightedUnits = highlightedUnitCount(
+    yearData,
+    selectedYear,
+    yearSelection,
+    selectedThread,
+  );
+
+  function trackSelectYear(year: string): void {
+    if (trackingData) {
+      const { subjectTitle, subjectSlug } = trackingData;
+      track.yearGroupSelected({
+        yearGroupName: year,
+        yearGroupSlug: year,
+        subjectTitle,
+        subjectSlug,
+        analyticsUseCase: analyticsUseCase,
+      });
+    }
+  }
+
+  const threadDef = (selectedThread: Thread["slug"]) =>
+    threadOptions.find((t) => t.slug === selectedThread);
+
+  return (
+    <Box
+      $position={["sticky", "static"]}
+      $display={["unset", "none"]}
+      $top={0}
+      $zIndex={"fixedHeader"}
+    >
+      <Box
+        $width={"100%"}
+        $background={"white"}
+        $mb={8}
+        data-test-id="filter-mobiles"
+      >
+        <Box>
+          <Box $dropShadow="mobileFilterSelector" $ph={[16, 0]} $pb={16}>
+            <Button
+              label="Highlight a thread"
+              icon="chevron-right"
+              $iconPosition="trailing"
+              variant="buttonStyledAsLink"
+              $mt={16}
+              onClick={onOpenModal}
+              data-testid="mobile-highlight-thread"
+            />
+            {selectedThread && (
+              <OakFlex>
+                <Box
+                  $textOverflow={"ellipsis"}
+                  $whiteSpace={"nowrap"}
+                  $overflow={"hidden"}
+                  data-testid="highlighted-threads-mobile"
+                  $maxWidth={"50%"}
+                >
+                  {threadDef(selectedThread)?.title}
+                </Box>
+                <Box $mh={6}> • </Box>
+                <Box data-testid="highlighted-units-box-mobile">
+                  <OakSpan aria-live="polite" aria-atomic="true">
```

Comment:
It was agreed with me and @assadk88 that this live region should stay where it is for now, because it ensures that the number of units highlighted is consistently announced to mobile screen reader users while the modal is open.

Reply:
I think we will need a follow-up to remove this, as I'm now getting consistent duplication of thread highlighting information when switching between threads in the modal. But that's for another PR I think.